packages/mobile-first-router/src/routerReducer.ts

```diff
@@ -200,3 +202,13 @@ const routerReducer: Function = (config: {
 };
 
 export default routerReducer;
+
+function getInitialTabQuery(config, initialRoute) {
```

Comment:
`const getInitialTabQuery = (config: MFNavigationConfig, initialRoute: string) => {`

Please use arrow syntax and types